docs/sources/storage/_index.md

```diff
@@ -64,6 +64,10 @@ DynamoDB is a cloud database offered by AWS. It is a good candidate for a manage
 
 DynamoDB is susceptible to rate limiting, particularly due to overconsuming what is called [provisioned capacity](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html). This can be controlled via the [provisioning](#provisioning) configs in the table manager.
 
+#### Costs and free tier
+
+Using Loki with DynamoDB will not fit in AWS free tier because it need far more read/write operations (ReadCapacityUnit-Hrs and WriteCapacityUnit-Hrs in AWS terms) that is covered by it. Free tier covers 18,600 Read and Write Capacity Unit-Hours per month. When using Loki with DynamoDB index you can expect about 750,000 Read and 450,000 Write Capacity Unit-Hours per month. These are highly approximate numbers because they are depend on a lot of factors, but you should consider monitoring your DynamoDB consumption to avoid unexpected costs.
```

Comment:
No sub heading is needed here for this extra paragraph.

I'm suggesting a much more vague wording, as there are many factors that affect the quantity of DynamoDB R/W ops. So, we can suggest that our users pay attention, but we don't want to tell them what to do.

```suggestion
Be aware that the AWS pricing structure is based on quantities of read and write operations (AWS terminology: ReadCapacityUnit-Hrs and WriteCapacityUnit-Hrs). Consider monitoring your DynamoDB
operations, such that you can tune your installation.
```